### PR TITLE
Minor design changes in the Sub Dialogues

### PR DESCRIPTION
**Pull request by Fidel365**
This PR concern a minor design change in the Sub Dialogue by centering the `ucrButtonSubDialogue  `

### CODE REVIEW

File: instat/sdgConstructRegexExpression.vb [hunk 1]
```diff
@@ -101,4 +101,8 @@ Public Class sdgConstructRegexExpression
     Private Sub ucrBaseSubdialog_ClickReturn(sender As Object, e As EventArgs) Handles ucrBaseSubdialog.ClickReturn
         ucrReceiverForRegex.AddtoCombobox(ucrReceiverForRegex.GetText)
     End Sub
+
+    Private Sub ucrBaseSubdialog_Load(sender As Object, e As EventArgs) Handles ucrBaseSubdialog.Load
+
```
Comment (N-thony) [inline instat/sdgConstructRegexExpression.vb:106]: can you remove this?

File: instat/sdgCombineGraphOptions.vb [hunk 2]
```diff
@@ -181,6 +181,10 @@ Public Class sdgCombineGraphOptions
     Private Sub sdgCombineGraphOptions_Load(sender As Object, e As EventArgs) Handles MyBase.Load
         LoadGraphs()
     End Sub
+
+    Private Sub ucrsdgLayoutBase_Load(sender As Object, e As EventArgs) Handles ucrsdgLayoutBase.Load
+
```
Comment (N-thony) [inline instat/sdgCombineGraphOptions.vb:186]: Remove?

**Comment by Fidel365** [comment]
@N-thony , I have made the changes

**Review by N-thony — APPROVED** [review]
@rdstern @lloyddewit this is just a small design changes in the sub dialogue. There is no issue related to it but just to enable Fidel to our usual routine.

**Review by lloyddewit — APPROVED** [review]
@Fidel365 Welcome to our team! Thank you for this PR.

@Fidel365 @N-thony this PR changes the size and position of 59 subdialogs. I assume that this is the intention so I approve.

@rdstern I guess it's enough for you to test just a small sample of these subdialogs to confirm that you're happy with the general approach.

Thanks

**Review by rdstern — APPROVED** [review]

**Pull request merged**